.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: python
 python:
   - "2.7"
+  - "3.6"
```

Comment:
I think you want to add this to `tox.ini`, to avoid the [Travis error](https://travis-ci.org/edx/django-oauth2-provider/jobs/282173094).

Reply:
Actually the problem was with Travis, because the tests run on my local machine just fine.
Anyway, the error seems to be solved now.

Reply:
Thank you.